client/app/iplb/home/updateQuota/iplb-update-quota.controller.js

```diff
@@ -7,14 +7,14 @@ class IpLoadBalancerUpdateQuotaCtrl {
         this.quota = Object.assign({}, quota);
 
         // Convert bytes to GiB
-        this.alert = this.quota.alert / Math.pow(1000, 3);
+        this.alert = this.quota.alert / (1000 ** 3);
```

Comment:
Does it actually works ? I know I suggested it but I'm not sure our configuration of BabelJS transpiles it correctly 😃

Reply:
Nice to know !